R/card.R

```diff
@@ -148,7 +148,7 @@ card_body <- function(..., fillable = TRUE, min_height = NULL, max_height = NULL
   }
 
   tag <- div(
-    class = "card-body",
+    class = "card-body bslib-gap-spacing",
     style = css(
```

Comment:
It probably makes sense to add a `padding` arg to `card_body()` in this PR (and use `validateCSSPadding()` to process it). 

(I was gonna add that, but since you're taking the reigns, I'll leave it to you 😄 )